### PR TITLE
Chapter 9

### PR DESCRIPTION
**Pull request by balqis277**
ready

### CODE REVIEW

**Comment by balqis277** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
       render :new
     end
   end
 
   def destroy
-    log_out
+    log_out if logged_in?
     redirect_to root_path
   end
+
```
Comment (anlvq-1058) [inline app/controllers/sessions_controller.rb:22]: remove this blank line

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**